_posts/2019-02-11-bootstrap-4-3-0.md

```diff
@@ -0,0 +1,64 @@
+- **Deprecated:** `img-retina`, `invisible`, `float`, and `size` mixins are now deprecated and will be removed in v5.
+
+Checkout the full [v4.3.0 ship list](https://github.com/twbs/bootstrap/issues/27893) and [GitHub project](https://github.com/twbs/bootstrap/projects/16) for the full details.
+
+[Head to to the v4.3.x docs](https://getbootstrap.com/docs/4.3/) to see the latest in action. The full release has been published to npm and will soon appear on the Bootstrap CDN and Rubygems.
+
+## Introducing responsive font sizes
+
+![Responsive font-sizes](/assets/img/2019/02/rfs.png)
+
+Our biggest new addition to Bootstrap in v4.3 is responsive font sizes, a [new project in the Bootstrap GitHub org](https://github.com/twbs/rfs) to automate calculate an appropriate `font-size` based on the dimensions of a visitor's device or browser viewport. Here's how it works:
+
+- All `font-size` properties have been switched to the `@include font-size()` mixin. Our Stylelint configuration now prevents the usage of `font-size` property.
+
+- Disabled by default, you can opt into this new behavior by toggling the `$enable-responsive-font-sizes` boolean variable.
+
+- `font-size`s are entirely configurable via Sass. Be sure to read the docs for how to modify the scales, variables, and more.
+
+While responsive font-sizes are disabled by default, we've enabled them in the custom CSS that powers our docs starting with v4.3. Please share feedback with us via GitHub issues or on Twitter. We've added some light guidance to [our Typography docs](https://getbootstrap.com/docs/4.3/content/typography/#responsive-font-sizes) to explain the feature. You can also learn more by [reading the rfs project documentation](https://github.com/twbs/rfs).
+
+## Open Collective
+
+Last December we launched our [Open Collective page](https://opencollective.com/bootstrap) with our [v3.4 release](https://blog.getbootstrap.com/2018/12/13/bootstrap-3-4-0/) to help support the maintainers contributing to Bootstrap. The team has been very excited about this as a way to be transparent about maintainer costs (both time and money), as well as recognition of efforts.
+
+## Branches, Hugo, and jQuery
+
+Right after shipping v4.3, we'll be tackling a few key changes on our road to active v5 development. These are larger changes to how we maintain and develop Bootstrap and are considered foundational for v5.
+
+- **Improving our branches for development.** `master` will become our new `v3-dev` branch. `v4-dev` will stay as-is, but we'll cut a new `master` branch from there to develop v5.
+
+- **We've moving to Hugo!** Jekyll has been great, but it's starting to slow us down in local development. We'll be making changes to our dependencies to support this move, and there's already a pull request in progress and near completion for the change. [Follow along](https://github.com/twbs/bootstrap/pull/28014) to see what's changing.
```

Comment:
Typo: It should be `We're moving..`

Reply:
Thanks, fixed inhttps://github.com/twbs/bootstrap-blog/commit/22402309bd28845560d7b71a0a0222bd6e7d0ab7